README.md

```diff
@@ -1,20 +1,63 @@
+We recommend using [Titanium](https://github.com/appcelerator/titanium) version 8.2.1.GA, or later. You'll also need to install [CocoaPods](http://cocoapods.org) if don't have it installed already:
+
+```bash
+$ [sudo] npm install -g titanium
+$ [sudo] gem install cocoapods
+```
+
+Now, clone the Appcelerator-iOS project to your local machine:
+
+```bash
+$ git clone https://github.com/PSPDFKit/Appcelerator-iOS.git
+```
+
+Once cloned, open the `Podfile` file on the root of the repository, and on line 6, replace `YOUR_COCOAPODS_KEY` with the private CocoaPods key for your copy of PSPDFKit, which you can retrieve from the [Customer Portal](https://customers.pspdfkit.com/customers/sign_in).
+
+Inside the Appcelerator-iOS project, run `pod install` :
+
+```bash
+$ cd Appcelerator-iOS/
+$ pod install
+```
+
+And now, build it: 
+
+```bash
+$ ti build -p ios --build-only
+
+    [DEBUG] Titanium SDK iOS directory: /Users/oscarcisneros/Library/Application Support/Titanium/mobilesdk/osx/8.2.1.GA/iphone
+    [INFO]  Project directory: /Users/oscarcisneros/Documents/PSPDFKit/Appcelerator-iOS
+    [INFO]  Module ID: com.pspdfkit
+    [INFO]  Module Type: Static Library (Objective-C)
+    [DEBUG] Writing module assets file: /Users/oscarcisneros/Documents/PSPDFKit/Appcelerator-iOS/Classes/ComPspdfkitModuleAssets.m
+    [DEBUG] Running: DEVELOPER_DIR=/Applications/Xcode.app/Contents/Developer /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -configuration Release -sdk iphonesimulator -UseNewBuildSystem=NO ONLY_ACTIVE_ARCH=NO clean build
+    [DEBUG] Running: DEVELOPER_DIR=/Applications/Xcode.app/Contents/Developer /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -configuration Release -sdk iphoneos -UseNewBuildSystem=NO ONLY_ACTIVE_ARCH=NO clean build
+    [INFO]  Creating universal library
+    [DEBUG] Searching library: /Users/oscarcisneros/Documents/PSPDFKit/Appcelerator-iOS/build/Release-iphoneos/libComPspdfkit.a
+    [DEBUG] Searching library: /Users/oscarcisneros/Documents/PSPDFKit/Appcelerator-iOS/build/Release-iphonesimulator/libComPspdfkit.a
+    [DEBUG] Running: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo /Users/oscarcisneros/Documents/PSPDFKit/Appcelerator-iOS/build/Release-iphoneos/libComPspdfkit.a /Users/oscarcisneros/Documents/PSPDFKit/Appcelerator-iOS/build/Release-iphonesimulator/libComPspdfkit.a -create -output /Users/oscarcisneros/Documents/PSPDFKit/Appcelerator-iOS/build/libcom.pspdfkit.a
+    [INFO]  Verifying universal library
+    [DEBUG] Running: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo -info /Users/oscarcisneros/Documents/PSPDFKit/Appcelerator-iOS/build/libcom.pspdfkit.a
+    [INFO]  Creating module zip
+    [INFO]  Writing module zip: /Users/oscarcisneros/Documents/PSPDFKit/Appcelerator-iOS/dist/com.pspdfkit-iphone-9.0.2.zip
+    [INFO]  Project built successfully in 9s 389ms
```

Comment:
I think the console output clutters the instructions, I would remove it altogether.